.github/workflows/CD.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: release to DockerHub
+on:
+  workflow_run:
+    workflows:
+      - CI
+    types:
+      - completed
+    branches:
+      - main
+    tags:
+      - '*'
```

Comment:
`tags` are unfortunately not supoorted by `workflow_run` :( 

References:

+ https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run